plugins/usbdmx/LibUsbThreadTest.cpp

```diff
@@ -87,6 +87,10 @@ void LibUsbThreadTest::testNonHotplug() {
 
 #if defined(LIBUSB_API_VERSION) && (LIBUSB_API_VERSION >= 0x01000102)
 void LibUsbThreadTest::testHotplug() {
+  if (!m_context) {
+    return;
```

Comment:
But there's two classes of skipping here, the #if, which means the code in olad won't run either and this check, which means the olad code should run if it's got access to /sys or whatever is blocked here. I just think we should warn where we've got a coverage gap and the tests aren't fully testing the code you're able to run in olad on that machine.